src/Sylius/Behat/Page/Admin/Product/FormTrait.php

```diff
@@ -206,13 +213,129 @@ public function getNumberOfAttributes(): int
     private function selectAttributeToBeAdded(string $attributeName): void
     {
         $driver = $this->getDriver();
-        $this->autocompleteHelper->select(
+        $this->autocompleteHelper->selectByName(
             $driver,
             $this->getElement('product_attribute_input')->getXpath(),
             $attributeName,
         );
     }
 
+    /*
+     * Media management
+     */
+
```

Comment:
```suggestion
```

Reply:
This class will disappear once unificated (form element), so let's skip it.